GTG/core/xml.py

```diff
@@ -72,6 +72,14 @@ def task_from_element(task, element: etree.Element):
     except (AttributeError, TypeError):
         pass
 
+    # Recurring tasks
+    try:
+        recurring = element.attrib['recurring']
```

Comment:
This throws a KeyError when the recurring attribute isn't present. Since it's not present for any tasks it would be better to use the .get() function that returns None when it can't find it.

`recurring = element.get('recurring')`